ddtrace/internal/datadog/profiling/stack_v2/src/stack_renderer.cpp

```diff
@@ -0,0 +1,89 @@
+#include "stack_renderer.hpp"
+#include "utf8_validate.hpp"
+
+using namespace Datadog;
+
+void
+StackRenderer::render_message(std::string_view msg)
+{
+    // This is unused for now, since the dd-trace-py profiler doesn't support emitting arbitrary messages.
+    (void)msg;
+}
+
+void
+StackRenderer::render_thread_begin(PyThreadState* tstate,
+                                   std::string_view name,
+                                   microsecond_t wall_time_us,
```

Comment:
We sometime use this type, and sometimes just int.  Is there a reason for which/when?